### PR TITLE
move util in a util folder with the sanitizer

### PR DESCRIPTION
**Pull request by Johann-S**
I hope @XhmikosR won't be angry against me but... I like to break everything and it's just the beginning 😈

### CODE REVIEW

**Review by Johann-S — COMMENTED** [review]
it generate a strange issue when you hide a collapse 🤔 
https://deploy-preview-28345--twbs-bootstrap.netlify.com/docs/4.3/components/collapse/#multiple-targets

File: js/src/collapse.js [hunk 1]
```diff
@@ -224,8 +233,6 @@ class Collapse {
 
     this._element.style[dimension] = `${this._element.getBoundingClientRect()[dimension]}px`
 
-    Util.reflow(this._element)
```
Comment (XhmikosR) [inline js/src/collapse.js]: Maybe this is the cause for the collapse issue?

Comment (Johann-S) [reply js/src/collapse.js]: No it was a little trick to force IE to refresh the UI

**Comment by Johann-S** [comment]
I fixed the collapse plugin 👍

**Review by XhmikosR — APPROVED** [review]

**Pull request merged**